torchio/data/image.py

```diff
@@ -117,11 +118,25 @@ def __init__(
             tensor: Optional[TypeData] = None,
             affine: Optional[TypeData] = None,
             check_nans: bool = False,  # removed by ITK by default
-            reader: Callable = read_image,
+            reader: Callable = None,
             **kwargs: Dict[str, Any],
             ):
         self.check_nans = check_nans
-        self.reader = reader
+
+        if reader is not None:
+            self.reader = reader
+        elif 'dims_bioformat' in kwargs:
+            from aicsimageio import AICSImage
+            dims = kwargs.pop('dims_bioformat')
+            bio_reader = lambda path: (
+                check_uint_to_int(
+                    AICSImage(path).get_image_dask_data(dims, **kwargs).compute()
```

Comment:
you may want to make this use the underlying reader object.

`AICSImage(path).reader.get_image_dask_data`

`AICSImage` dumps some dimensions if they aren't part of the standard set and I don't know what the intention of this library is.